code/script.js

```diff
@@ -231,95 +241,145 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  setSecret()
+  generateBoard()
+  count = 0
```

Comment:
Since count is declared as a global variable at line 216 I dont think we need to declare its value again